samples/test/util.py

```diff
@@ -224,6 +240,28 @@ def _create_run():
     fire.Fire(main)
 
 
+def run_v2_pipeline(
```

Comment:
Currently the sdk side has not integrated with v2 mode, right?

Reply:
Yes, the v2 backend compiler CLI is build just for testing. In full v2 architecture, it should be part of api server.